File: Java/UberVan.java
import java.util.ArrayList
import java.util.Map

class UberVan extends Car{
    Map<String, ArrayList<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberX(String license, Account drive, Map<String, 
                 ArrayList<String, Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(license,drive);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, drive,typeCarAccepted,seatsMaterial):
        super.__init__(license, drive)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial